bitnami/airflow/README.md

```diff
@@ -383,6 +548,82 @@ Find more information about how to deal with common errors related to Bitnami’
 
 ## Notable changes
 
+### 7.0.0
```

Comment:
Since we are also upgrading the major of PostgreSQL and Redis, we should also include how to upgrade those.

Reply:
Done in my last commit